### ISSUE TITLE
Design GUI for steering wheel knob area 

### ISSUE THREAD

**Issue by stefanieschreiner** [i315315/amos2022ws02-automotive-test-app#21]
## User story
1.	As a User
2.	I want to be able to enter an area in the test app where all use cases for simulating steering wheel buttons are collected 
3.	So that I can easily switch between the use cases.
With this User story the graphical user interface of the steering wheel knob area of the app is to be designed.

## Acceptance criteria
* The graphical user interface of the steering knob area in the app is designed.
* An XML for the GUI of the steering knob area in the app has been created in Android studio.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
